tests/checkpointing/test_model_checkpoint.py

```diff
@@ -462,12 +472,43 @@ def test_default_checkpoint_behavior(tmpdir):
 
 
 def test_ckpt_metric_names_results(tmpdir):
-    model = EvalModelTemplate()
-    model.training_step = model.training_step_result_obj
+    class LessBoringModel(BoringModel):
```

Comment:
this is a bit confusing again the same model name

Reply:
Changed